packages/grid/_modules_/grid/components/cell/GridEditInputCell.tsx

```diff
@@ -20,6 +21,19 @@ const useUtilityClasses = (ownerState: OwnerState) => {
   return composeClasses(slots, getDataGridUtilityClass, classes);
 };
 
+const InputBaseStyled = styled(InputBase, {
```

Comment:
@mui-org/x Do we agree on this convention? The styled components are always named to match the parent component name?

```suggestion
const GridEditInputCellRoot = styled(InputBase, {
```

Reply:
I'm ok with this. I used `Styled` as that seemed to be the naming convention used in the core but I can see how what you are suggesting also makes sense in our case. I've updated this and all the other places where we had this name construct.